.github/workflows/database-migrations.yml

```diff
@@ -32,6 +36,8 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.version }}
```

Comment:
Rather than checking out inputs.version, we want instead to pass in IMAGE_TAG to `make release-run-database-migrations`

Reply:
I was a little worried about backwards incompatibilities, but I guess you're running the workflow from `main` anyway (reflecting current deploy processes), and if something about migrations/deploying the app has changed in an incompatible way with older commits, you'll have to manually intervene anyway.

So can update for this.